### PR TITLE
doc/releases/mimic: upgrade steps

### PR DESCRIPTION
**Pull request by liewegas**
Signed-off-by: Sage Weil <sage@redhat.com>

### CODE REVIEW

**Comment by liewegas** [comment]
jenkins render docs

**Comment by liewegas** [comment]
@batrick are there cephfs-specific upgrade steps to add to this?

File: doc/releases/mimic.rst [hunk 1]
```diff
@@ -6,8 +6,137 @@ this is not the final release yet, consider testing this on your testing and non
...
+     # ceph osd versions
+     {
+        "ceph version 12.2.5 (...) luminous (stable)": 12,
+        "ceph version 13.2.0 (...) mimic (stable)": 22,
+     }
+
+#. Upgrade all CephFS MDS daemons by upgrading packages and restarting
+   daemons on all hosts::
+
+     # systemctl restart ceph-mds.target
```
Comment (batrick) [inline doc/releases/mimic.rst]: Point them at: https://github.com/ceph/ceph/blob/8937f5f41a2a05bc3ffd8f719409cd4053287efe/doc/cephfs/upgrading.rst#upgrading-the-mds-cluster

**Comment by liewegas** [comment]
jenkins render docs

File: doc/releases/mimic.rst [hunk 2]
```diff
@@ -245,7 +397,7 @@ Notable Changes
 
 
 Other Notable Changes
-~~~~~~~~~~~~~~~~~~~~~
+---------------------
```
Comment (joscollin) [inline doc/releases/mimic.rst]: This doesn't seem to make any difference in the rendered doc. I found the same - *no effect* in other docs too.

File: doc/releases/mimic.rst [hunk 3]
```diff
@@ -6,8 +6,162 @@ this is not the final release yet, consider testing this on your testing and non
 production clusters for a preview of upcoming features and please report all the
 bugs to the tracker.
 
+Major Changes from Luminous
+---------------------------
+
+- *Dashbaord*:
```
Comment (LenzGr) [inline doc/releases/mimic.rst]: @liewegas too late, as this PR has already been merged, but there's typo here: "Dashbaord".

I also added a blurb about the new Dashboard to the PendingReleaseNotes document, maybe that provides some additional color.

**Pull request merged**